database/tables/user_sessions.pg

```diff
@@ -3,6 +3,7 @@ columns
     data: jsonb not null
     expires_at: timestamp with time zone not null
     id: bigint not null default nextval('user_sessions_id_seq'::regclass)
+    revoked_at: timestamp with time zone
```

Comment:
My inclination is to go with consistency and use `deleted_at`, but I don't feel strongly enough to fight about it. Just in general I prefer to the lower mental overhead of not needing to remember that sessions are revoked, assessments are deleted, etc. To me the meaning of "deleting" a session is pretty clearly the same as "revoking" it.

In general I'm in favor of standardizing all tables to have columns `id`, `created_at`, and (when appropriate) `deleted_at`. That being said, we already violate this in multiple places so using `revoked_at` here isn't any major problem.

Reply:
I'm going to stick with `revoked_at` for now. A realization I had while thinking about this: I believe we can safely hard-delete expired sessions where `user_id IS NULL`. These sessions are created only by unauthenticated users (bots, probably) and don't hold any useful information for us. Given that we at least in principle will be hard-deleting from this table, I think avoiding `deleted_at` is a good idea.